source/VirtualDesktop/VirtualDesktop.csproj

```diff
@@ -68,6 +68,36 @@
     <DebugType>pdbonly</DebugType>
     <DebugSymbols>true</DebugSymbols>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'ReleaseStore|AnyCPU'">

```

Comment:
What's the point of including these changes in your pull request?

Reply:
There is no point, I forgot to make a branch and moved on after you didn't read the PR